pip_audit/_dependency_source/requirement.py

```diff
@@ -89,6 +94,20 @@ def collect(self) -> Iterator[Dependency]:
                 reqs: List[InstallRequirement] = []
                 req_names: Set[str] = set()
                 for req in rf.requirements:
+                    if req.req is None:
```

Comment:
NTS: We need a changelog entry for this fix. I can commandeer the branch to add that, once we confirm that the fix is good.

Reply:
[9abac3a](https://github.com/trailofbits/pip-audit/pull/331/commits/9abac3a03f750c7c3a35bf7308492b447e54675f)